src/external_cli/cargo/install.rs

```diff
@@ -21,6 +21,7 @@ fn is_installed(program: &str) -> bool {
 pub(crate) fn if_needed(
     program: &str,
     package: &str,
+    package_version: Option<&str>,
     ask_user: bool,
     hidden: bool,
 ) -> anyhow::Result<bool> {
```

Comment:
How does this handle when the user has `wasm-bindgen-cli` already installed, but it's the wrong version? Does it replace the current version?

Reply:
'cargo install' replaces the already existing version. I don't think there exists a good solution to keep the old binary yet.

Found this:
https://github.com/rust-lang/cargo/issues/12366
https://github.com/rust-lang/cargo/issues/9778

Reply:
I think if the version already exists, but the version doesn't match, we need to ask the user if they want to overwrite the version.

Not ideal, but the user will get an error if the version doesn't match, so this is the only way for them to proceed with the compilation.

The `-f` flag can be used to force install the new version.

Reply:
`-f` is only needed if the version we would want to install is already present. 

```
❯ cargo install  wasm-bindgen-cli --version 0.2.70
❯ wasm-bindgen --version
wasm-bindgen 0.2.70
❯ cargo install  wasm-bindgen-cli --version 0.2.99
❯ wasm-bindgen --version
wasm-bindgen 0.2.99
```